pkg/engine/runtime/kubernetes/kubernetes_runtime.go

```diff
@@ -724,6 +764,31 @@ func getNamedGVK(gvk schema.GroupVersionKind) schema.GroupVersionKind {
 	}
 }
 
+// Get the amount of dependent resources of the owner
+func getDependentAmount(object *unstructured.Unstructured) int {
```

Comment:
For the default amount of `Deployment`, `StatefulSet` and `CronJob`, maybe `1` is better than `-1` because there should be only one pod owned by these resources if `replicas` or `jobTemplate` are not declared.

Reply:
> For the default amount of `Deployment`, `StatefulSet` and `CronJob`, maybe `1` is better than `-1` because there should be only one pod owned by these resources if `replicas` or `jobTemplate` are not declared.
Good catch. Thanks!